src/Builder/CliMenuBuilder.php

```diff
@@ -216,6 +232,14 @@ public function addSubMenuFromBuilder(string $text, CliMenuBuilder $builder) : s
             $menu->setStyle($this->menu->getStyle());
         }
 
+        if (!$menu->getCheckboxStyle()->hasChangedFromDefaults()) {
+            $menu->setCheckboxStyle(clone $this->menu->getCheckboxStyle());
+        }
+
+        if (!$menu->getRadioStyle()->hasChangedFromDefaults()) {
+            $menu->setRadioStyle(clone $this->menu->getRadioStyle());
+        }
```

Comment:
We could extract this to a private method, since we do it below as well. But we can do that later.

Reply:
Above even.